crates/egui/src/containers/resize.rs

```diff
@@ -252,7 +252,12 @@ impl Resize {
         }
 
         if let Some(user_requested_size) = user_requested_size {
-            state.desired_size = user_requested_size;
+            if self.resizable.x {
+                state.desired_size.x = user_requested_size.x;
+            }
+            if self.resizable.y {
+                state.desired_size.y = user_requested_size.y;
+            }
```

Comment:
AFAICT this code path is only hit by `Window`, which should already respect `resizable`. Is this really changing any behavior in practice? Or is the PR just about demonstrating an existing behavior in the demo?